### ISSUE TITLE
multigpu

### ISSUE THREAD

**Issue by nationalflag** [eriklindernoren/PyTorch-YOLOv3#14]
It seems can't use multigpu?

**Comment by eriklindernoren** [comment]
Support for multigpu is on the todo-list.

**Comment by weiaicunzai** [comment]
@eriklindernoren Sorry to ask, do we have support for multigpu now? Thanks

**Comment by LeoCHANGcn** [comment]
just add
```python
model =  torch.nn.DataParallel(model)
```
and run again.

**Comment by Lausannen** [comment]
@LeoCHANGcn It seems that multi-gpus version can not accelerate the training. I compared single GPU and multi gpus, one epoch costed similiar time. During your test, did you have this problem?

**Comment by dtmoodie** [comment]
I have also noticed the same issue.

**Comment by tensmyo** [comment]
@LeoCHANGcn
I added the following at Initiate model, 
> @train.py
> model = Darknet(opt.model_config_path)
> model =  torch.nn.DataParallel(model)
but new error on loss.backward()
> grad can be implicitly created only for scalar outputs
does loss function should be change too?

**Comment by Robmosh91** [comment]
Hi,
this issue is coming up since loss.backward()is equivalent to loss.backward(torch.Tensor([1])). 
When using DataParallel this is invalid. Instead you should use either
loss.backward(torch.Tensor([1, 1])) or loss.sum().backward(). 
The behaviour of both should be the exact same.

**Comment by ujsyehao** [comment]
you can refer to https://github.com/ujsyehao/yolov3-multigpu

**Comment by stevenalexis** [comment]
@Robmosh91 Hi,when I used your suggestion,thefollowing issue occurred:'DataParallel' object has no attribute 'losses'?
